Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/Format.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats
+{
+    public enum Format
```

Comment:
@skendrot the Reason for the Formatter class, is that different formats require different sets of buttons. E.g. RichText supports Underline, while Markdown doesn't. Therefore, RichText has an underline button and Markdown doesn't. This also allows users to define their own Formatter class with their own styling and Default Button map.

Perhaps the abstract methods can be removed, and attached to button directly, when someone adds a CommonButton to their ButtonMap?